velox/exec/PartitionedOutputBufferManager.h

```diff
@@ -261,6 +252,15 @@ class PartitionedOutputBufferManager {
   static std::weak_ptr<PartitionedOutputBufferManager> getInstance(
       const std::string& host = "local");
 
+  std::unique_ptr<OutputStreamListener> newChecksumListener() const {
```

Comment:
Checksum in the name suggests that the listener can only be used for calculating a checksum, but this is not necessarily the case. Let's rename to remove "checksum". Please, add comments to explain these methods.